### PR TITLE
Adds 5 new drinks

### PR DESCRIPTION
**Pull request by ghost**
:cl: NewSta
add: Added 5 new drinks: Alexander, Between the Sheets, Kamikaze, Mojito and Sidecar.
add: Added Menthol and Sake to the drink dispensers.
/:cl:

[why]: # 

Added five new IBA Official Cocktails for a better bartender experience. With new sprites, of course. A truly great new feature.

### CODE REVIEW

**Comment by Nabski89** [comment]
Hey, what about my mint! What kind of mojito is this.

**Comment by europax** [comment]
K.

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -632,3 +632,33 @@
...
+	name = "Kamikaze"
+	id = "kamikaze"
+	results = list("kamikaze" = 3)
+	required_reagents = list("vodka" = 1, "triple_sec" = 1, "limejuice" = 1)
+
+/datum/chemical_reaction/mojito
+	name = "Mojito"
+	id = "mojito"
+	results = list("mojito" = 5)
+	required_reagents = list("rum" = 1, "sugar" = 1, "limejuice" = 1, "sodawater" = 1, "creme_de_menthe" = 1)
```
Comment (ShizCalev) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: >mojito
>creme de menthe
>iba
motherfucker what

atleast use actual mint, or minttoxin
/obj/item/reagent_containers/food/snacks/mint

**Comment by europax** [comment]
I'll try to fix a few more things because of this implementation, wait a bit

**Comment by europax** [comment]
All good! Ready to merge.

One thing though - we might need to edit #37598 afterwards due to a merge conflict that would happen

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**